tests/foreman/longrun/test_inc_updates.py

```diff
@@ -128,100 +147,66 @@ def setUpClass(cls):
         rhel_6_partial_ak = ActivationKey(
-            name=u'rhel_6_partial_ak',
-            content_view=cls.rhel_6_partial_cv,
-            host_collection=[cls.rhel_6_partial_hc],
+            name=gen_alpha(),
+            content_view=self.rhel_6_partial_cv,
+            host_collection=[self.rhel_6_partial_hc],
             **kwargs
         ).create()
 
-        # Step 14: Assign subscriptions to activation keys
-        # Fetch available subscriptions
-        subs = Subscription(organization=cls.org).search()
+        # Assign subscription to activation key. Fetch available subscriptions
+        subs = Subscription(organization=self.org).search()
         assert len(subs) > 0
 
-        # Add subscriptions to activation key
+        # Add subscription to activation key
         sub_found = False
         for sub in subs:
-            if sub.read_json()['product_name'] == u'Employee SKU':
-                for act_key in [rhel_6_ak, rhel_6_partial_ak]:
-                    act_key.add_subscriptions(data={
-                        u'subscription_id': sub.id
-                    })
+            if sub.read_json()['product_name'] == DEFAULT_SUBSCRIPTION_NAME:
+                rhel_6_partial_ak.add_subscriptions(data={
+                    u'subscription_id': sub.id
+                })
                 sub_found = True
         assert sub_found
 
-        # Step 15: Enable product content in activation keys
-        for act_key in [rhel_6_ak, rhel_6_partial_ak]:
-            act_key.content_override(data={'content_override': {
-                u'content_label': PRD_SETS['rhel6_sat6tools']['label'],
-                u'value': u'1'
-            }})
+        # Enable product content in activation key
+        rhel_6_partial_ak.content_override(data={'content_override': {
+            u'content_label': REPOS['rhst6']['id'],
+            u'value': u'1'
+        }})
 
-        # Step 16: Create three client machines and register them to satellite
-        # Register the first vm with rhel_6_ak and the other two vms with
+        # Create client machine and register it to satellite with
         # rhel_6_partial_ak
-        cls.vm = [
-            VirtualMachine(distro='rhel67', tag='incupdate')
-            for _ in range(3)
-        ]
-        cls.setup_vm(cls.vm[0], rhel_6_ak.name, cls.org.label)
-        for i in range(1, 3):
-            cls.setup_vm(cls.vm[i], rhel_6_partial_ak.name, cls.org.label)
+        self.vm = VirtualMachine(distro='rhel67', tag='incupdate')
+        self.setup_vm(self.vm, rhel_6_partial_ak.name, self.org.label)
+        self.vm.enable_repo(REPOS['rhva6']['id'])
+        self.vm.run('yum install -y {0}'.format(REAL_0_RH_PACKAGE))
 
         # Find the content hosts and save them
-        hosts = Host(organization=cls.org).search()
-        cls.hosts = []
-        cls.partial_hosts = []
-
+        hosts = Host(organization=str(self.org.id)).search()
```

Comment:
`organization` accepting not organization entity or organization id (numeric), but a string containing organization id.